NEWS.md

```diff
@@ -91,6 +91,8 @@ Library improvements
     * Rank one update and downdate functions, `lowrankupdate`, `lowrankupdate!`, `lowrankdowndate`,
     and `lowrankdowndate!`, for dense Cholesky factorizations ([#14243],[#14424])
 
+    * All `sparse` methods now retain provided numerical zeros as structural nonzeros.
```

Comment:
+1 for exporting `dropzeros!` and making other operations retain zeros (including the `sparse` function).